webapp/components/Hashtag/Hashtag.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <ds-tag>#{{ id }}</ds-tag>
+  <ds-tag>
+    <nuxt-link :to="hashtagUrl">#{{ id }}</nuxt-link>
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-31T17:06:55Z" title="Thursday, October 31st 2019, 6:06:55 pm +01:00">Oct 31, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,5 +1,7 @@
 <template>
-  <ds-tag>#{{ id }}</ds-tag>
+  <ds-tag>
+    <a v-bind:href="`/?hashtag=${id}`" target="_blank">#{{ id }}</a>
```


```suggestion
    <nuxt-link :to="hashtagUrl">#{{ id }}</nuxt-link>
```
and implement a `hashtagUrl` computed property :wink: